docker/Dockerfile.hf_oob

```diff
@@ -0,0 +1,115 @@
+# syntax=docker/dockerfile:1
+ARG BASE_IMAGE=ubuntu:22.04
+FROM ${BASE_IMAGE} AS dev-base
+RUN apt-get update && \
+        DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+        bc \
+        build-essential \
+        ca-certificates \
+        ccache \
+        cmake \
+        curl \
+        ffmpeg \
+        git \
+        libtcmalloc-minimal4 \
+        libjpeg-dev \
+        libpng-dev \
+        libsm6 \
+        libxext6 \
+        numactl \
+        pybind11-dev \
+        wget \
+        vim \
+    && rm -rf /var/lib/apt/lists/*
+RUN /usr/sbin/update-ccache-symlinks
+RUN mkdir /opt/ccache && ccache --set-config=cache_dir=/opt/ccache
+ENV PATH /opt/conda/bin:$PATH
+
+FROM dev-base AS conda
+ARG PYTHON_VERSION=3.8
+RUN curl -fsSL -v -k -o ~/miniconda.sh -O https://repo.anaconda.com/miniconda/Miniconda3-py38_23.9.0-0-Linux-x86_64.sh && \
+    chmod +x ~/miniconda.sh && \
+    # Fix conda install issue
+    sed -i "s;#!/bin/sh;#!/bin/bash;" ~/miniconda.sh && \
+    ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda config --set channel_priority strict && \
+    /opt/conda/bin/conda config --append channels conda-forge && \
+    /opt/conda/bin/conda install -y python=${PYTHON_VERSION} \
+        astunparse \
+        cffi \
+        cython \
+        cmake>=3.13.0 \
+        dataclasses \
+        future \
+        git-lfs \
+        intel-openmp \
+        ipython \
+        mkl \
+        mkl-include \
+        ninja \
+        numpy \
+        requests \
+        typing \
+        typing_extensions \
+        Pillow \
+        pkg-config \
+        pybind11 \
+        pyyaml \
+        setuptools \
+        jemalloc \
+        openssl \      
+        weasyprint && \
+    /opt/conda/bin/conda init && \
+    /opt/conda/bin/conda clean -ya && \
+    # Fix GLIBCXX version issue
+    rm -f /opt/conda/lib/libstdc++.so.6
+
+FROM dev-base AS build
+COPY --from=conda /opt/conda /opt/conda
+COPY --from=conda /root/.bashrc /root/.bashrc
```

Comment:
Why we need copy it?

Reply:
Cuz hf_testcase rely on `${CONDA_PREFIX}/lib/libiomp5.so` and `jemalloc.so`

Reply:
please set the right ${CONDA_PREFIX} before run the hf_testcase script, or we can modify the hf_testcase scripts like https://github.com/chuanqi129/inductor-tools/blob/main/scripts/modelbench/inductor_test.sh#L1. It will use `$(dirname $(which conda))/../` as default ${CONDA_PREFIX}